File: BMI_Tipper/app/src/main/java/com/example/tipper/RecipeActivity.java
package com.example.tipper;

import androidx.appcompat.app.AppCompatActivity;
import android.widget.TextView;
import android.os.Bundle;

public class RecipeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_recipe);

        TextView recipeCacio = findViewById(R.id.recipe_1);
        TextView recipeKedgeree = findViewById(R.id.recipe_2);
    }
}

File: BMI_Tipper/app/src/main/java/com/example/tipper/RecipeActivity.kt
package com.example.tipper

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class RecipeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_recipe)

        val recipeCacio = findViewById<TextView>(R.id.recipe_1)
        val recipeKedgeree = findViewById<TextView>(R.id.recipe_2)
    }
}